packages/cli/src/types.ts

```diff
@@ -28,6 +28,7 @@ export interface FarmCLIBuildOptions {
   sourcemap?: boolean;
   minify?: boolean;
   treeShaking?: boolean;
+  format?: 'cjs' | 'esm';
```

Comment:
`format` should be added to `output`, thanks!

Reply:
this is the type of cli It should be right.